src/helpers.js

```diff
@@ -0,0 +1,47 @@
+export const getStoredCredentials = (keys) => {
+    console.log(`Getting values for keys: ${JSON.stringify(keys)}`);
+
+    return keys.map(key => new Promise((res) => {
+        chrome.storage.sync.get(key, (result) => {
+            res(result);
+        });
+    }));
+}
+
+export const sleep = (milliseconds = 1500) => {
+    return new Promise((res) => {
+        setTimeout(() => {
+            res();
+        }, milliseconds);
+    })
+}
+
+export const login = (username, password) => {
+    fetch("https://sdarot.buzz/login", {
```

Comment:
we need to pass this function a baseURL of the current URL for this to work across multiple sites url.